Gems/GradientSignal/Code/Tests/GradientSignalTestFixtures.cpp

```diff
@@ -115,6 +168,108 @@ namespace UnitTest
+    {
+        AZ_PROFILE_FUNCTION(Entity);
+
+        // All components are created, so activate the entity
+        ActivateEntity(m_testEntity.get());
+
+        // Create a gradient sampler and run through a series of points to see if they match expectations.
+        GradientSignal::GradientSampler gradientSampler;
+        gradientSampler.m_gradientId = m_testEntity->GetId();
+
+        // Get the height and width ranges for querying from our benchmark parameters
+        float height = aznumeric_cast<float>(state.range(0));
+        float width = aznumeric_cast<float>(state.range(1));
+        int64_t totalQueryPoints = state.range(0) * state.range(1);
+
+        // Call GetValues() for every height and width in our ranges.
+        for (auto _ : state)
+        {
+            // Set up our vector of query positions.
+            AZStd::vector<AZ::Vector3> positions(totalQueryPoints);
+            size_t index = 0;
+            for (float y = 0.0f; y < height; y += 1.0f)
+            {
+                for (float x = 0.0f; x < width; x += 1.0f)
+                {
+                    positions[index++] = AZ::Vector3(x, y, 0.0f);
+                }
+            }
+
+            // Query and get the results.
+            AZStd::vector<float> results(totalQueryPoints);
+            gradientSampler.GetValues(positions, results);
+        }
+    }
+
+    void GradientSignalBenchmarkFixture::RunEBusGetValueBenchmark(benchmark::State& state)
+    {
+        AZ_PROFILE_FUNCTION(Entity);
+
+        // All components are created, so activate the entity
+        ActivateEntity(m_testEntity.get());
+
+        GradientSignal::GradientSampleParams params;
+
+        // Get the height and width ranges for querying from our benchmark parameters
+        float height = aznumeric_cast<float>(state.range(0));
+        float width = aznumeric_cast<float>(state.range(1));
+
+        // Call GetValue() for every height and width in our ranges.
+        for (auto _ : state)
+        {
+            for (float y = 0.0f; y < height; y += 1.0f)
+            {
+                for (float x = 0.0f; x < width; x += 1.0f)
+                {
+                    float value = 0.0f;
+                    params.m_position = AZ::Vector3(x, y, 0.0f);
+                    GradientSignal::GradientRequestBus::EventResult(
+                        value, m_testEntity->GetId(), &GradientSignal::GradientRequestBus::Events::GetValue, params);
+                    benchmark::DoNotOptimize(value);
+                }
+            }
+        }
+    }
+
+    void GradientSignalBenchmarkFixture::RunEBusGetValuesBenchmark(benchmark::State& state)
+    {
+        AZ_PROFILE_FUNCTION(Entity);
+
+        // All components are created, so activate the entity
+        ActivateEntity(m_testEntity.get());
+
+        // Create a gradient sampler and run through a series of points to see if they match expectations.
```

Comment:
super nit: I'm not really sure what this comment means by matching expectations. You're not really checking for values, just measuring performance.

Reply:
Whoops, stale comment.  I originally wrote this code as a unit test to verify that GetValues and GetValue are producing the same results.  In that context, the comment made sense. :)  Removing the comment.